benchmarks/nel/requirements.txt

```diff
@@ -1,4 +1,7 @@
 pyyaml
 requests
 tqdm
-prettytable
\ No newline at end of file
+prettytable
+scikit-learn
+thefuzz[speedup]
```

Comment:
Where is this library used? I see `fuzzyset` but not this. 

It also looks like `fuzzyset` (or [fuzzyset2](https://github.com/axiak/fuzzyset) I guess?) uses `python-levenshtein`, which has maintenance issues. It might be a good idea to submit a PR to change that to use `rapidfuzz`.

Reply:
Was a mistake, replaced it with `fuzzyset2`. One advantage of `fuzzyset` over `rapidfuzz` is that it implements not only fuzzy string matching, but also an index structure for reverse n-gram lookups (similar to https://microsoft.github.io/spacy-ann-linker IIRC). If the issue is just concerns about maintenance, I'd stick with `fuzzyset` for the benchmark and the time being.